### DISCUSSION TITLE
Modify Wallet page and Wallet Setting page confusion

### DISCUSSION THREAD

**Discussion by pavlenex** [btcpayserver/btcpayserver#2276] [General]
With #2164 merged, we've introduced a new Modify Wallet page which in essence is very similar to the Wallet Settings page. However, they do have different information.

I'm opening this discussion to discuss on how we can either distinguish them or unify those pages. Because, right now the flow is quite confusing.
- What is the difference between the two?
- What's the goal of having a modify wallet page?
- What's the goal of having wallet settings page?
- What kind of information are we trying to show with them to the user?

![Screenshot from 2021-02-11 13-20-31](https://user-images.githubusercontent.com/36959754/107636057-5f88ca00-6c6c-11eb-9144-ee14428d618d.png)

![Screenshot from 2021-02-11 13-20-49](https://user-images.githubusercontent.com/36959754/107636067-644d7e00-6c6c-11eb-97c4-ac7b7d9b431d.png)

Ping @dennisreimann @dstrukt @NicolasDorier

**Comment by dstrukt** [comment]
Edit:

From an overall UX perspective, I think we should try to move everything into the updated UI, or all into the "settings" tabs so as to not have both a settings / wallet pill with their own set of options, which I know is why this ticket was flagged (excellent work), just clarifying.

1. The "label" is something I think that should be part of the flow, which was addressed with the "naming a wallet" concept, despite the two being slightly different. It might be nice to carry this over, and potentially rename it, but keep the same functionality as lightly discussed in the past. 

2. The derivation scheme is redundant and could be solely left in the updated flow.

3. The "account" section is something that I think should be discussed, but similarly to #2, that whole section could be removed from this view and added to the updated flow.
a) The "account key" could be added above the fingerprint on the updated flow.
b) The "master" OR "root" fingerprint wording should be decided upon.
c) There's still a little bit of uncertainty around the scan function for me.

4. The "other actions" should be surfaced, and we should try to do this with all screens moving forward. Even if those "additional options" are tucked behind an "advanced settings" dropdown for the. The question now becomes, do we include this actions as "advanced settings" on the Modify wallet view?

Old: Taking a look that this morning, along with the icons (was hoping to get to those last night, but hopefully will have something to share with @dennisreimann + discuss tomorrow morning)

**Reply by dstrukt** [reply]
Follow up:

@dennisreimann and I discussed a few things, and believe there is another discussion needed to proceed with this, but many of the options that are tucked into the "other options" should indeed be pulled out, but they might have a better fit functionally / contextually, elsewhere.

I would propose this is a ticket we discuss at the next design meeting @pavlenex. I've taken note as well.

**Reply by pavlenex** [reply]
Sounds good, let's def discuss this at the next design meeting, will schedule something soon.
